Readme.md

```diff
@@ -0,0 +1,55 @@
+# KOB Email Verification Service
+##  Abstract
+ *  KOB Email Verification Services proves the ownership of user over his email. It provides verified digital credential for user’s email which is stored in user’s wallet. User     submit this digital credential, when he provides his email for availing third party service.
```

Comment:
- "KOB Email Verification Services" -> KOB Email Verification Service.
- Please do check the alignment if necessary.
- You can take out the bullet point as theres just a single paragraph.